Config/dbConnect.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+
+const dbConnect = async() => {
```

Comment:
Make sure you leave some proper spacing everywhere, Like in 
```suggestion
const dbConnect = async () => {
```
everywhere in the try catch block. Just an improvement for better readability